diff --git a/dub.sdl b/dub.sdl
index 023dd4e..f50fb61 100644
--- a/dub.sdl
+++ b/dub.sdl
@@ -44,7 +44,7 @@ configuration "static_dynamicCRT" {
     
     // Linux x86_64
     preBuildCommands \
-        "cmake -DSTATIC_CIMGUI= -S deps -B deps/build_linux_x64_cimguiStatic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -DSTATIC_CIMGUI= -S deps -B deps/build_linux_x64_cimguiStatic" \
         "cmake --build deps/build_linux_x64_cimguiStatic --config Release" platform="linux-x86_64"
 
     sourceFiles "$PACKAGE_DIR/libs/x86_64/linux/Static/cimgui.a" platform="linux-x86_64"
@@ -52,7 +52,7 @@ configuration "static_dynamicCRT" {
 
     // Linux aarch64
     preBuildCommands \
-        "cmake -DSTATIC_CIMGUI= -S deps -B deps/build_linux_aarch64_cimguiStatic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -DSTATIC_CIMGUI= -S deps -B deps/build_linux_aarch64_cimguiStatic" \
         "cmake --build deps/build_linux_aarch64_cimguiStatic --config Release" platform="linux-aarch64"
 
     sourceFiles "$PACKAGE_DIR/libs/aarch64/linux/Static/cimgui.a" platform="linux-aarch64"
@@ -94,7 +94,7 @@ configuration "static_staticCRT" {
 
     // Linux x86_64
     preBuildCommands \
-        "cmake -DSTATIC_CIMGUI= -S deps -B deps/build_linux_x64_cimguiStatic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -DSTATIC_CIMGUI= -S deps -B deps/build_linux_x64_cimguiStatic" \
         "cmake --build deps/build_linux_x64_cimguiStatic --config Release" platform="linux-x86_64"
 
     sourceFiles "$PACKAGE_DIR/libs/x86_64/linux/Static/cimgui.a" platform="linux-x86_64"
@@ -102,7 +102,7 @@ configuration "static_staticCRT" {
 
     // Linux aarch64
     preBuildCommands \
-        "cmake -DSTATIC_CIMGUI= -S deps -B deps/build_linux_aarch64_cimguiStatic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -DSTATIC_CIMGUI= -S deps -B deps/build_linux_aarch64_cimguiStatic" \
         "cmake --build deps/build_linux_aarch64_cimguiStatic --config Release" platform="linux-aarch64"
 
     sourceFiles "$PACKAGE_DIR/libs/aarch64/linux/Static/cimgui.a" platform="linux-aarch64"
@@ -153,14 +153,14 @@ configuration "dynamic_dynamicCRT" {
 
     // Linux x86_64
     preBuildCommands \
-        "cmake -S deps -B deps/build_linux_x64_cimguiDynamic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -S deps -B deps/build_linux_x64_cimguiDynamic" \
         "cmake --build deps/build_linux_x64_cimguiDynamic --config Release" platform="linux-x86_64"
 
     copyFiles "libs/x86_64/linux/Dynamic/cimgui.so" platform="linux-x86_64"
 
     // Linux aarch64
     preBuildCommands \
-        "cmake -S deps -B deps/build_linux_aarch64_cimguiDynamic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -S deps -B deps/build_linux_aarch64_cimguiDynamic" \
         "cmake --build deps/build_linux_aarch64_cimguiDynamic --config Release" platform="linux-aarch64"
 
     copyFiles "libs/aarch64/linux/Dynamic/cimgui.so" platform="linux-aarch64"
@@ -213,14 +213,14 @@ configuration "dynamic_staticCRT" {
 
     // Linux x86_64
     preBuildCommands \
-        "cmake -S deps -B deps/build_linux_x64_cimguiDynamic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -S deps -B deps/build_linux_x64_cimguiDynamic" \
         "cmake --build deps/build_linux_x64_cimguiDynamic --config Release" platform="linux-x86_64"
         
     copyFiles "libs/x86_64/linux/Dynamic/cimgui.so" platform="linux-x86_64"
 
     // Linux aarch64
     preBuildCommands \
-        "cmake -S deps -B deps/build_linux_aarch64_cimguiDynamic" \
+        "cmake -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -S deps -B deps/build_linux_aarch64_cimguiDynamic" \
         "cmake --build deps/build_linux_aarch64_cimguiDynamic --config Release" platform="linux-aarch64"
 
     copyFiles "libs/aarch64/linux/Dynamic/cimgui.so" platform="linux-aarch64"
